src/api/Models/Borehole.cs

```diff
@@ -0,0 +1,472 @@
+﻿using NetTopologySuite.Geometries;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BDMS.Models;
+
+/// <summary>
+/// Represents a borehole entity in the database.
+/// </summary>
+[Table("borehole")]
+public class Borehole
+{
+    /// <summary>
+    /// Gets or sets the <see cref="Borehole"/> id.
+    /// </summary>
+    [Column("id_bho")]
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="User"/> who created the <see cref="Borehole"/>.
+    /// </summary>
+    [Column("created_by_bho")]
+    public int? CreatorId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="User"/> who created the <see cref="Borehole"/>.
+    /// </summary>
+    public User? Creator { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Borehole"/> contact id.
+    /// </summary>
+    [Column("contact_id")]
+    public int? ContactId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Borehole"/> created date.
+    /// </summary>
+    [Column("created_bho")]
+    public DateTime? Created { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Borehole"/> updated date.
+    /// </summary>
+    [Column("updated_bho")]
+    public DateTime? Updated { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="User"/> who updated the <see cref="Borehole"/>.
+    /// </summary>
+    [Column("updated_by_bho")]
+    public int? UpdaterId { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="User"/> who updated the <see cref="Borehole"/>.
+    /// </summary>
+    public User? Updater { get; set; }
```

Comment:
PP: `Updated` als Begriff benutzen, also `UpdatedById` und `UpdatedBy`. So wird das gleiche Schema wie für `LockedBy` verwendet.

Reply:
Es ist etwas verwirrend, die Spaltennamen heissen je nach Tabelle updater oder updatedBy. Bzw.  creator, createdBy oder author. 

Habe versucht das über die Tabellen hinweg etwas zu vereinheitlichen. Aber ich kann es auch überall auf UpdatedBy umstellen. 

"Locker" ist umgekehrt ja nicht so sinnvoll ;) .

Reply:
Ah, alles klar. Dann kannst du es so lassen wenn du willst. Ich habe halt einfach die Spaltennamen pro Tabelle angeschaut und fand es verwirrend, wenn hier Updater steht und in der DB-Tabelle `updated_by`.